src/components/organisms/UserProfileModal/UserProfileModal.scss

```diff
@@ -65,4 +61,51 @@ $page-max-width: 1240px;
       grid-row: 1;
     }
   }
+
+  .visits {
```

Comment:
nit: wondering if these classes will be used by non-badge visits too, should they be named as such? It's OK if not.